processors/pubmed_unregistered_trials/processor.py

```
@@ -0,0 +1,70 @@
+        unregistered_trial = extractors['extract_trial'](record)
+        conn['database'].begin()
+        try:
+
+            # Create/Update unregistered trial from the publication
+            if not publication['registry_ids']:
+                trial_id, is_primary = base.writers.write_trial(
+                    conn, unregistered_trial, source_id, record['meta_id']
+                )
+                base.writers.write_record(
+                    conn, record, source_id, trial_id, unregistered_trial, is_primary
+                )
+
+            # Publication is not/no longer an unregistered trial but a results document
+            else:
+                delete_record = "DELETE FROM records WHERE identifiers @> '%s'"
+                delete_record = delete_record % json.dumps(unregistered_trial['identifiers'])
+                conn['database'].query(delete_record)
+                for identifiers in publication['registry_ids']:
+                    trial = base.helpers.find_trial_by_identifiers(
+                        conn, identifiers=identifiers
+                    )
+                    if trial:
+                        doc_category_id = extractors['extract_document_category'](record)
+                        document = extractors['extract_document'](record)
+                        document.update({
+                            'source_id': source_id,
+                            'document_category_id': doc_category_id,
+                        })
+                        document_id = base.writers.write_document(conn, document)
+                        base.writers.write_trial_relationship(
+                            conn, 'document', document, document_id, trial['id']
+                        )
+        except sqlalchemy_exceptions.DBAPIError:
+            conn['database'].rollback()
+            base.config.SENTRY.captureException(extra={
+                'record': record,
+            })
```

I agree with swallowing exceptions on the `data_contributions` processors, but why do the same here? Swallowing exceptions have the bad side-effect that our processors will pass on Airflow, so it won't notify us, retry or stop the next steps.